src/http/request.cr

```diff
@@ -121,6 +117,89 @@ class HTTP::Request
     BadRequest.new
   end
 
+  private METHODS = %w(GET HEAD POST PUT DELETE CONNECT OPTIONS PATCH TRACE)
```

Comment:
Is this faster as a tuple? (could allow LLVM to optimize the loop since size is known?)

Reply:
Someone else commented it. I tried it and there's no difference. I also said before that tuples shouldn't be used for everything, just for returning multiple things from methods.